src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
+let operator = false;
+let operatorWhich = "";
+let firstTime = false;
+let dotClicked = false;
+
+const updateTheme = (a) => {
+	themeValue = a.target.value;	
+}
+
+const number = (x) => {
+	
+	if (operator) {
+		value = x;
+	} else if(value.toString().length == 14) {
+		return;
+	} else{
+			if (dotClicked) {
+				let valueLength = value.length;
+				value = Number(value) + x*(Math.pow(10, -(counter)));				
```

Comment:
Some spaces needed in this line. I suggest to use some linter like eslint.
There are also other places that needs spaces, please check all of them.

Reply:
changed